requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+coverage==4.4.1
```

Comment:
Don't add all output from `pip freeze`. Please add only packages that you installed with `pip install`.